src/gui/symbols/area_symbol_settings.cpp

```diff
@@ -267,6 +267,8 @@ AreaSymbolSettings::AreaSymbolSettings(AreaSymbol* symbol, SymbolSettingDialog*
 		emit propertiesModified();
```

Comment:
I don't see how this patch fits into the Custom Icon series. Should it live in a different branch?

Reply:
I could apply this commit separately, but at the moment it is needed only for the following commit which adds another tab.

Reply:
I see. Please consider mentioninig in patch comment that this is pre-requesite for the next change.